src/scripting/lua_widget_methods.cpp

```diff
@@ -463,14 +486,15 @@ int luaW_open(lua_State* L)
 	lk.add_log("Adding widgets module...\n");
 	static luaL_Reg const gui_callbacks[] = {
 		//TODO: the naming is a bit arbitrary: widgets with different
-		//      types of elements use add_node, eidgets with only
+		//      types of elements use add_node, widgets with only
 		//      one type of element use add_element
 		{ "add_item_of_type",   &intf_add_item_of_type },
 		{ "add_item",           &intf_add_dialog_item },
 		{ "focus",              &intf_set_dialog_focus },
 		{ "set_canvas",         &intf_set_dialog_canvas },
 		{ "set_callback",       &intf_set_dialog_callback },
-		{ "remove_items_at",     &intf_remove_dialog_item },
+		{ "remove_items_at",    &intf_remove_dialog_item },
```

Comment:
Why is there a change to this line? I also can't see what the change is… probably whitespace changed?

Reply:
whitespace

Reply:
I'd prefer you don't do this kind of thing in future, but since it's only one line in a small PR, I'll let it pass this time.